include/platform/radio.h

```diff
@@ -207,7 +262,8 @@ void otPlatRadioSetShortAddress(otInstance *aInstance, uint16_t aShortAddress);
  * @retval ::kThreadError_None     Successfully enabled.
  * @retval ::kThreadError_Failure  The radio could not be enabled.
  */
-ThreadError otPlatRadioEnable(otInstance *aInstance);
+ThreadError otPlatRadioEnable(otInstance *aInstance, otPlatRadioReceiveDone receiveCallback,
+                              otPlatRadioTransmitDone transmitCallback);
```

Comment:
My vote would be to add otPlatRadioSetDelegate instead, which is passed a struct of function pointers.
That minimizes changes to existing PHY api and allows either the app or the stack itself to call enable / disable without disrupting the callback linkage.